.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @heroku/languages
+* @Uvalente
```

Comment:
Do we need a codeowners file do you think? If we do, maybe we should add the QA team as approvers instead?

Reply:
I don't expect this repo to have many PRs, so we could keep the QA team as approvers without adding any real load on them.